### PR TITLE
Implicit scheme of BGRa creep model

### PR DESCRIPTION
**Pull request by wenqing**
This PR presents the implementation of the Implicit scheme of BGRa creep model (see https://www.opengeosys.org/docs/benchmarks/creepbgra/creepbrga/)

Two new benchmarks for the implementation have been added (ufz/ogs5-benchmarks#29
).

### CODE REVIEW

**Comment by wenqing** [comment]
@norihiro-w Do you have any comment?

File: FEM/fem_ele_vec.cpp [hunk 1]
```diff
@@ -1954,10 +1950,8 @@ void CFiniteElementVec::LocalAssembly_continuum(const int update)
         JFNK = true;
 #endif
     //
-    ThermalExpansion = 0.0;
-    // Thermal effect
-    if (smat->Thermal_Expansion() > 0.0)
-        ThermalExpansion = smat->Thermal_Expansion();
+    const double ThermalExpansion = (smat->Thermal_Expansion() > 0.0) ?
```
Comment (norihiro-w) [inline FEM/fem_ele_vec.cpp]: - in which case, one has a negative thermal expansion coeff? 
- If the coeff can be negative, we can make it shorter  with `ThermalExpansion = std::max(smat->Thermal_Expansion(), 0.0);`

File: FEM/fem_ele_vec.h [hunk 2]
```diff
@@ -19,12 +19,20 @@
...
 #include "fem_ele.h"
 #include "FEMEnums.h"
+#include "Material/Solid/BGRaCreep.h"
 
 namespace SolidProp
 {
 class CSolidProperties;
 }
 
+namespace MaterialLib
```
Comment (norihiro-w) [inline FEM/fem_ele_vec.h]: i guess these lines are not needed since line 22 includes the header.

File: FEM/fem_ele_vec.cpp (hunk 1 above)
Comment (wenqing) [reply FEM/fem_ele_vec.cpp]: Removed this ternary assignment.

The only place for that case is for output the solid parameters, see
https://github.com/ufz/ogs5/blob/9ca6118ccb09973140626d17ecbb3b06b96c3d17/FEM/rf_msp_new.cpp#L8569

**Comment by wenqing** [comment]
@norihiro-w  Thanks!

**Comment by norihiro-w** [comment]
:+1:

**Pull request merged**